features/admin/add_buyer_domain.feature

```diff
@@ -0,0 +1,30 @@
+@add-buyer-domain
+Feature: Admin buyer domains
+
+Scenario: Admin user can log in and add a buyer domain
```

Comment:
This doesn't really describe the scenario that follows - where a user **can't** add a domain because it already exists.  It would be nice to have a separate step where a new domain is successfully added... maybe <a long-ish random string>.com would work for the success scenario?

Reply:
Changed the wording here